src/main/java/uk/nhs/ctp/tkwvalidation/AuditDispatcher.java

```diff
@@ -0,0 +1,79 @@
+package uk.nhs.ctp.tkwvalidation;
+
+import ca.uhn.fhir.context.FhirContext;
+import java.net.URI;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.Base64;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.hl7.fhir.dstu3.model.OperationOutcome;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.RequestEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import uk.nhs.ctp.tkwvalidation.model.AuditMetadata;
+import uk.nhs.ctp.tkwvalidation.model.AuditMetadata.Headers;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class AuditDispatcher {
+  private final FhirContext fhirContext;
+  private final RestTemplate restTemplate;
+
+  @Value("${reports.validation.server}")
+  private String reportValidationServer;
+
+  public String getValidationUrl() {
+    return reportValidationServer + "/$evaluate";
+  }
+
+  public String dispatchToTkw(byte[] zipData, AuditMetadata zipMetadata) {
+
+    var base64Zip = Base64.getEncoder().encode(zipData);
+    var validatorUrl = URI.create(getValidationUrl());
+    var interactionDate = zipMetadata.getInteractionDate()
+        .atOffset(ZoneOffset.UTC)
+        .format(DateTimeFormatter.ISO_INSTANT);
+
+    var request = RequestEntity.post(validatorUrl)
+        .header(HttpHeaders.ACCEPT, "application/fhir+json")
+        .header(HttpHeaders.CONTENT_TYPE, "application/zip")
+        .header("Content-Transfer-Encoding", "base64")
+        .header(Headers.SUPPLIER_ID, zipMetadata.getSupplierId())
+//        .header(Headers.API_VERSION, zipMetadata.getApiVersion().getVersion())
```

Comment:
is this line to be enabled in 400 also?

Reply:
Nope, sorry, the PR wasn't yet complete - I had raised it in order to deploy and test it on staging, it's added now!